CDP4WebServices.API/config_debug.json

```diff
@@ -16,8 +16,8 @@
     "DatabaseManage": "cdp4manage",
     "StatementTimeout": 180,
     "LogSqlCommands": false,
-    "IsDbSeedEnabled": true,
-    "IsDbRestoreEnabled": true
+    "IsDbSeedEnabled": false,
```

Comment:
Was this intentional?

Reply:
Yes, because this has overwritten my local test database once and I don't want that to happen again.

Reply:
there is a "test" config that should be used which has these set to `true`

Reply:
That's why I set them to false here, in the "debug" config.